src/components/MovieDetails.tsx

```diff
@@ -1,9 +1,127 @@
-const MovieDetails = () => {
+import { useState, useEffect, ReactNode } from 'react';
+
+import api from 'src/service/api';
+
+import ReadMore from './ReadMore';
+
+import '../styles/MovieDetails.scss';
+
+enum ImageWidth {
+  w500
+}
+
+type MovieDetailsProps = {
+  match: { params: { id: string } }
+}
+
+type MovieInfo = {
+  poster_path: string;
+  original_title: string;
+  title: string;
+  tagline: string;
+  status: string;
+  release_date: string;
+  budget: string;
+  popularity: string;
+  vote_average: string;
+  runtime: string;
+  genres: [{ id: null, name: string }],
```

Comment:
id null? probably should be number or string

Reply:
changed to number | null as it comes with type number .